tests/e2e/cucumber/features/smoke/link.feature

```diff
@@ -53,3 +53,24 @@ Feature: link
     And "Anonymous" should not be able to open the old link "myPublicLink"
     And "Anonymous" logs out
     And "Alice" logs out
+
+
+  Scenario: Quick link
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+    And "Alice" creates the following folders in personal space using API
+      | name         |
+      | folderPublic |
+    And "Alice" creates the following files into personal space using API
+      | pathToFile             | content     |
+      | folderPublic/lorem.txt | lorem ipsum |
+    When "Alice" logs in
+    And "Alice" opens the "files" app
+    And "Alice" copies quick link of the resource "folderPublic" from the context menu
+    When "Anonymous" opens the public link "Quicklink"
```

Comment:
```suggestion
    And "Anonymous" opens the public link "Quicklink"
```

Reply:
This should be the when step I'll change the above one